dune-project

```diff
@@ -1,2 +1,22 @@
-(lang dune 1.1)
+(lang dune 1.10)
```

Comment:
What do you think of using 1.11 here? We would get the better build command in the opam file.